SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -41,8 +41,6 @@ class FSpatialGDKEditorToolbarModule : public IModuleInterface, public FTickable
 		RETURN_QUICK_DECLARE_CYCLE_STAT(FSpatialGDKEditorToolbarModule, STATGROUP_Tickables);
 	}
 
-	FSimpleMulticastDelegate OnSpatialShutdown;
```

Comment:
Any reason to remove this delegate? Seems like it could be useful going forwards.

Reply:
Removing it since it's not used anywhere, I would rather add it when it is necessary.

Reply:
I don't think we should be removing this as customers might want to bind to it :D

Reply:
I don't think we should be removing this as customers might want to bind to it :D